server/src/main/webapp/WEB-INF/rails/webpack/views/pages/roles/policy_widget.tsx

```diff
@@ -0,0 +1,252 @@
+/*
+ * Copyright 2019 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {MithrilViewComponent} from "jsx/mithril-component";
+import _ from "lodash";
+import m from "mithril";
+import Stream from "mithril/stream";
+import {Directive, GoCDRole, PluginRole, Policy} from "models/roles/roles";
+import * as Buttons from "views/components/buttons";
+import {FlashMessage, MessageType} from "views/components/flash_message";
+import {AutocompleteField, SuggestionProvider} from "views/components/forms/autocomplete";
+import {SelectField, SelectFieldOptions} from "views/components/forms/input_fields";
+import {Table} from "views/components/table";
+import styles from "./index.scss";
+
+interface PolicyAttrs {
+  role: GoCDRole | PluginRole;
+}
+
+export class PolicyWidget extends MithrilViewComponent<PolicyAttrs> {
+  static headers() {
+    return [
+      "Permission"
+      , "Action"
+      , "Type"
+      , "Resource"
```

Comment:
indentation?

Reply:
Can u pls explain a bit more?

Reply:
Moved the `,` to the same line as value.